### ISSUE TITLE
Bug: wxPanel SetMinSize and SetMaxSize wrong code

### ISSUE THREAD

**Issue by hobbitdev** [KeyWorksRW/wxUiEditor#851]
### Description:
Is there something I don't understand at all
Are there two wrong rows with syntax error
![image](https://user-images.githubusercontent.com/33904565/205143145-6dacbd28-76c2-4525-b6dd-d1b80acf4c39.png)

**Comment by KeyWorksRW** [comment]
Looks like all the wxWindow form versions need a code review of min/max size settings:

- Help text refers to smart_size property which was dropped
- In a Dialog, Frame and Panel, XRC expects the settings to be part of the top sizer rather then the wxWindow inherited class.
- Dialog generation is ignoring the dialog units setting
- Frame is generating it as SetSizeHints
- Panel is setting it in two different places and neither one is correct
- Wizard ignores it entirely

Not that the top level sizer has a minimum size property, but the code it generates ignores dialog units. I suspect, given what the XRC code expects that sizer takes precedence.

Bottom line is that this needs a deep dive into the wxWidgets wxWindow code to be sure we know what it does and doesn't accept, and then make the code generation consisten across all the wxWindow forms.

**Comment by Randalphwa** [comment]
The XRC code is a problem since both the Dialog and Panel can have a minimum size set _and_ it's top level sizer can have a minimum size set as well. These could be different values since the sizer won't include the parent window's frame/title size. I suspect the real issue is with the XRC code in wxWidgets not having a way to set the form's min/max size. Applying the form's settings will _not_ be the same as it would in C++/Python generated code or our own mock-up. As such, I am not including and XRC changes in the PR that fixes the other items.
